### ISSUE TITLE
box-sizing in base css

### ISSUE THREAD

**Issue by allanesquina** [pure-css/pure#522]
Hi, do you agree that box-sizing: border-box should be in base.css file? 
It's repeated in some files. I think it's important for other elements like menu as well.
Maybe adding this code in the base.css file and removing from the others.

``` css
*, *:before, *:after {
    -webkit-box-sizing: border-box;
    -moz-box-sizing: border-box;
    box-sizing: border-box;
}
```

Good reference:
https://css-tricks.com/box-sizing/

:+1:

**Comment by mattconvente** [comment]
I agree. I found this article a while ago which convinced me to add it as a wildcard. It's 3 1/2 years old, but the points are still valid. http://www.paulirish.com/2012/box-sizing-border-box-ftw/

**Comment by mattconvente** [comment]
@allanesquina I submitted a PR that address this issue: https://github.com/yahoo/pure/pull/529

**Comment by redonkulus** [comment]
We recently released v2.0.0 but did not include this change. If anyone wants to test this out we could pull this in since the release is still new but would be risky. Going to close this for now unless there is a huge demand.
